python/lsst/sims/maf/stackers/generalStackers.py

```diff
@@ -397,3 +399,45 @@ def _run(self, simData, cols_present=False):
         simData['season'] = season
         return simData
 
+
+class OpSimFieldStacker(BaseStacker):
+    """Add the OpSim fieldID for each RA/Dec pointing.
+
+    Parameters
+    ----------
+    raCol : str, opt
+        Name of the RA column. Default fieldRA.
+    decCol : str, opt
+        Name of the Dec column. Default fieldDec.
+
+    """
+    def __init__(self, raCol='fieldRA', decCol='fieldDec', degrees=True):
+        self.colsReq = [raCol, decCol]
+        self.colsAdded = ['fieldID']
+        self.units = ['radians', 'radians']
+        self.raCol = raCol
+        self.decCol = decCol
+        self.degrees = degrees
+        fields = getOpSimField()
```

Comment:
It may be helpful to leave a comment here about whether the field information returned by getOpsimField is in degrees or radians?